source/_posts/pruning-1.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How to Prune deep CNN based architectures? 
```

Comment:
Add 3 more keyword from this list.

NAYAN
Traffic monitoring
AI ---
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING